src/hello-world/arithmetic.md

```diff
@@ -0,0 +1,39 @@
+---
+minutes: 5
+---
+
+# Arithmetic
+
+```rust,editable
+fn main() {
+    let a: i32 = 12;
+    let b: i32 = 35;
+    let c: i32 = 37;
+    if a.pow(2) + b.pow(2) == c.pow(2) {
```

Comment:
I think it's better not to introduce methods here if we're trying to focus on arithmetic (where the built-in operators give us plenty to teach in one go, and various interesting math methods like `sin` and `count_leading_zeros` don't quite fall under arithmetic).